app/workers/publish_classification_event_worker.rb

```diff
@@ -12,7 +12,7 @@ def perform(classification_id)
     if classification.complete?
       EventStream.push("classification",
         event_id: "classification-#{classification.id}",
-        event_time: "#{classification.updated_at}",
+        event_time: classification.updated_at.iso8601,
```

Comment:
Yeah - i didn't like it either but i wanted to use `a_hash_including` spec which fails when passing a datetime object, I'll rework it.